docs/how-to-guides/advanced-user-guides/enable-experimental-features/sqlite-caching.md

```diff
@@ -0,0 +1,42 @@
+---
+title: SQLite-backed caching
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-experimental-features/sqlite-caching"/>
+</head>
+
+:::caution
+Enabling the SQLite-backed caching of Kubernetes objects is a **highly experimental** feature, it is not yet officially supported in Rancher. It is not intended for any production use. SUSE customers should consult SUSE Support before activating this feature.
+:::
+
+
+## About this Feature
+This feature enables an optional cache of Kubernetes objects with the aim of improving performance, especially while browsing tabular data in the UI with thousands of rows. It also provides improved UI sorting, filtering and pagination features. 
+
+## Enabling instructions
+To enable the feature, perform the following steps:
+1. Log into the Rancher UI.
+1. In the upper left corner, click **☰ > Global Settings > Feature Flags**.
+1. Go to **`ui-sql-cache`** and click **⋮ > Activate > Activate**.
+1. Wait for Rancher to restart. This also restarts agents on all downstream clusters.
+1. In the upper left corner, click **☰ > Global Settings > Performance**.
+1. Go to **Server-side Pagination** and check the **Server-side Pagination** option.
+1. Click **Apply**
+1. Refresh the page (`CTRL + R`)
```

Comment:
```suggestion
## Enabling SQLite-Backed Caching

1. In the upper left corner, click **☰ > Global Settings > Feature Flags**.
1. Find **`ui-sql-cache`** and select **⋮ > Activate > Activate**.
1. Wait for Rancher to restart. This also restarts agents on all downstream clusters.
1. In the upper left corner, click **☰ > Global Settings > Performance**.
1. Go to **Server-side Pagination** and check the **Server-side Pagination** option.
1. Click **Apply**.
1. Refresh the page (`CTRL + R`).
```

Is there no button to refresh the page? If not, different OS's have different combinations of key presses to perform a refresh and I guess we'd have to list them out

Reply:
I addressed the refresh hotkey question in afe50d3d2a5